src/stream/consumer-stream.ts

```diff
@@ -44,14 +44,15 @@ async function* genPoll(
 
 
 
-const singleConsumerStream = (config: ClientConfig) => async (
+export const singleConsumerStream = (config: ClientConfig) => async (
   poll: PollMessages,
-  interval: 1000
+  // interval: 1000
 ): Promise<Readable> => {
   const c = await rawClientGetter(config);
   if (!c.isAuthenticated)
     await c.authenticate(config.credentials);
   const ps = Readable.from(genPoll(c, poll), { objectMode: true });
+  // const ps = Readable.from(genAutoCommitedPoll(c, poll, interval), { objectMode: true });
```

Comment:
Is it expected too?

Reply:
sorry i'm still playing r&d with those stream